pages/about.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/style.css" rel="stylesheet">
+ 
 </head>
 <body>
-  
-</body>
+<section class = "flex-container">
+<h1><b><i><u>Who is She?</u></i></b></h1>
+
+<h2>
+<p><em>Somone's Mom</em></p>
```

Comment:
`p` tags are not aloud in heading tags, and causes the page not to validate.

Reply:
`em` is a tag with valid semantic meaning, but it should be used to express that you are emphasizing a particular word/phrase in a larger passage, not just for styling purposes.  From MDN:
>The `<em>` element is for words that have a stressed emphasis compared to surrounding text, which is often limited to a word or words of a sentence and affects the meaning of the sentence itself.